From a8a0b89593edcda28fc5d41d9e97c5f9adec6258 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Wed, 07 Oct 2020 22:19:23 +0300
Subject: [PATCH] [SW-21702] goya2: remove unwanted register settings

This patch removes unwanted register programming done as part of
PCIe link bringup sequence.

Change-Id: If731b323c917b6bdfb07590b24d72c69fcb3fa17
---

diff --git a/goya2/common.c b/goya2/common.c
index 99a7345..41a0d1e 100644
--- a/goya2/common.c
+++ b/goya2/common.c
@@ -84,17 +84,10 @@
 	 */
 	regmap_writel(1, mmPCIE_PHY_CR_PARA_SEL_0);
 	regmap_writel(1, mmPCIE_PHY_CR_PARA_SEL_1);
-	regmap_writel(1, mmPCIE_PHY_CR_PARA_SEL_2);
-	regmap_writel(1, mmPCIE_PHY_CR_PARA_SEL_3);
 
 	/* Select indirect access to CR PARA I/F instead of PRBI */
 	regmap_writel(1, mmPCIE_PHY_CR_PARA_CFG);
 
-	/*
-	 * Use PHY configuration from SoC online
-	 * instead of default values
-	 */
-	regmap_writel(1, mmPCIE_PHY_PHY_EXT_CTRL_SEL);
 	preboot_init_pcie_phy_fw_update();
 }
 
